data/strings.yml

```diff
@@ -331,8 +331,10 @@ code_signing:
         delete: "Delete"
     expose:
       is_expose: Expose for Pull Request
+    file_storage_id: File Storage ID
+    download_url: Download URL
     upload_action: Upload file (max. <maximum_file_size_in_megabytes> MB)
-    upload_instructions: Click here or Drag & Drop to upload
+    upload_instructions: Add a <strong>File Storage ID</strong> before uploading a file
```

Comment:
Pls remove this item from line 338.
**no_files_added_yet: No files added yet.**